docs/2.0. APIs.md

```diff
@@ -72,8 +72,22 @@ Server Metadata.
 
 ## API Interaction
 
+The following diagrams highlight the different sequences of interactions that occur when accessing a Protected
+Resource depending on the OAuth 2.0 grant type used.
+
+### Authorization Code Grant Flow
+
+This flow can be used for any NMOS interaction.
+
 ![Authorization Flow](images/authorization_flow.png)
 
+### Client Credentials Grant Flow
+
+This flow can only be used in specific interactions. Refer to [BCP-003-02][BCP-003-02] for permitted uses of
```

Comment:
I only see Auth Code in BCP-003-02 https://specs.amwa.tv/bcp-003-02/branches/v1.0-dev/docs/1.0._Authorization_Practice.html#authorization-grant-constraints

Could this have a more specific link?

Reply:
The heading "Authorization Grant Constraints" is a bit ambiguous really, as that section talks about all "Authorization Grants" not specifically the "Auth _Code_ Grant" - if anything that heading should be called "Client Credentials Grant Constraints" as it details the specific instances of when to use the CC Grant. Either way I think referencing that BCP is the correct doc